internal/compose/helm.go

```diff
@@ -117,6 +117,10 @@ func installHelmRelease(name string, release *cfg.Release) {
 		args = append(args, "--skip-crds")
 	}
 
+	if release.Wait {
+		args = append(args, "--wait")
+	}
+
```

Comment:
One more thing, can you move this block down below the `Timeout` block to keep the order in line with the struct fields

Reply:
Yeah make more sense 👍